source/ed/connectors/gtfs_parser.cpp

```diff
@@ -388,85 +388,113 @@ bool StopsGtfsHandler::parse_common_data(const csv_row& row, T* stop) {
     return true;
 }
 
-StopsGtfsHandler::stop_point_and_area StopsGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
+nm::StopArea* StopsGtfsHandler::build_stop_area(Data& data, const csv_row& row) {
+    nm::StopArea* sa = new nm::StopArea();
+    if (!parse_common_data(row, sa)) {
+        delete sa;  // don't forget to free the data
+        return nullptr;
+    }
+
+    if (has_col(wheelchair_c, row) && row[wheelchair_c] == "1") {
+        sa->set_property(navitia::type::hasProperties::WHEELCHAIR_BOARDING);
+    }
+    gtfs_data.stop_area_map[sa->uri] = sa;
+    data.stop_areas.push_back(sa);
+
+    if (has_col(timezone_c, row)) {
+        auto tz_name = row[timezone_c];
+        sa->time_zone_with_name = gtfs_data.tz.get_tz(tz_name);
+    }
+    if (!sa->time_zone_with_name.second) {
+        // if no timezone has been found, we set the default one
+        sa->time_zone_with_name = {data.tz_wrapper.tz_name, data.tz_wrapper.boost_timezone};
+    }
+    if (has_col(desc_c, row)) {
+        add_gtfs_comment(gtfs_data, data, sa, row[desc_c]);
+    }
+    return sa;
+}
+
+nm::StopPoint* StopsGtfsHandler::build_stop_point(Data& data, const csv_row& row) {
+    nm::StopPoint* sp = new nm::StopPoint();
```

Comment:
same, can you use std::unique_ptr?

Reply:
I think it should be done in another PR, or even better, use smart ptr everywhere